docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+| Image - Generate a service file from an original file (image_generate_a_service_file_from_an_original_file) | generate_image_derivative |use the houdini queue to generate an "image" media of type "image/jpeg" tagged with ServiceFile, from the original file, and save it in the public filesystem in a folder for the year and month.  |
+| Video - Generate a thumbnail from an original file (video_generate_a_thumbnail_from_an_original_file) | generate_image_derivative | Use the **homarus** queue to generate an image media tagged Thumbnail of type image/jpeg from the original file by taking 1 frame from the 1 second timestamp, and save it to the public filesystem in a folder for the year and month. |
+| Video - Generate a service file from an original file (video_generate_a_service_file_from_an_original_file) | generate_video_derivative |use the Homarus queue to generate a video media tagged service file of type video/mp4 from an original file, save it to the public filesystem in a folder for the year and month.  |
+
+
+## Migrations
+
+
+| Name | 
+|---|
+|Tags migration for islandora_defaults feature (islandora_defaults_tags)| 
+
+Contents of the islandora_defaults_tags migration: 
+
+|Vocabulary|Term name|Description|External Uri|
+|---|---|---|---|
+|islandora_display|"Open Seadragon"|"Display using the Open Seadragon viewer"|http://openseadragon.github.io|
+|islandora_display|"PDFjs"|"Display using the PDF.js viewer"|http://mozilla.github.io/pdf.js|
+|resource_types|"Collection"|"An aggregation of resources"|http://purl.org/dc/dcmitype/Collection|
+|resource_types|"Dataset"|"Data encoded in a defined structure"|http://purl.org/dc/dcmitype/Dataset|
+|resource_types|"Image"|"A visual representation other than text"|http://purl.org/dc/dcmitype/Image|
+|resource_types|"Interactive Resource"|"A resource requiring interaction from the user to be understood, executed, or experienced"|http://purl.org/dc/dcmitype/InteractiveResource|
+|resource_types|"Moving Image"|"A series of visual representations imparting an impression of motion when shown in succession"|http://purl.org/dc/dcmitype/MovingImage|
+|resource_types|"Physical Object"|"An inanimate, three-dimensional object or substance"|http://purl.org/dc/dcmitype/PhysicalObject|
+|resource_types|"Service"|"A system that provides one or more functions"|http://purl.org/dc/dcmitype/Service|
+|resource_types|"Sound"|"A resource primarily intended to be heard"|http://purl.org/dc/dcmitype/Sound|
+|resource_types|"Still Image"|"A static visual representation"|http://purl.org/dc/dcmitype/StillImage|
+|resource_types|"Software"|"A computer program in source or compiled form"|http://purl.org/dc/dcmitype/Software|
+|resource_types|"Text"|"A resource consisting primarily of words for reading"|http://purl.org/dc/dcmitype/Text|
+
+
+core.base_field_override.node.islandora_object.changed.yml
```

Comment:
I dunno about these things.  They're just auto-provided by Drupal because we configure these fields settings.